extension/src/state_aggregate/accessors.rs

```diff
@@ -0,0 +1,495 @@
+use crate::{
+    accessors::{
+        AccessorIntoIntValues, AccessorIntoValues, AccessorStateIntTimeline, AccessorStateTimeline,
+    },
+    datum_utils::interval_to_ms,
+    raw::{Interval, TimestampTz},
+    state_aggregate::*,
+};
+
+#[pg_operator(immutable, parallel_safe)]
+#[opname(->)]
+pub fn arrow_state_agg_into_values<'a>(
```

Comment:
I did find it more useful to group these functions with the non-arrow versions.  I'm not strictly opposed to having them in a separate file like this, but I don think it's important that we're consistent with how we organize our code.